README.md

```diff
@@ -98,6 +96,6 @@ Making the following change reduces startup time exponentially, and gets rid of
 
 ## Authors
 
-Mike Bradley  
-Noi Sek  
-Tiffany Zhang  
+Mike Bradley
```

Comment:
By the way, these are alphabetized, but if anyone wants to reorder them I'd have no objection.